setup.sh

```diff
@@ -1,8 +1,19 @@
 wd=$(pwd -P)
 
+# https://stackoverflow.com/questions/226703/how-do-i-prompt-for-input-in-a-linux-shell-script
+yesno () {
+    echo $1
```

Comment:
I actually like the first option on the top answer there better.
Then this can become
```
read -p "$1 [y/n]" yn
case yn in
  [Yy]* ) return 0; break;;
  [Nn]* ) return 1;;
esac